From 458286f8ff0638e553dd031892f63f4bdadce552 Mon Sep 17 00:00:00 2001
From: freedesktop_sdk_updater <freedesktop_sdk_updater@libreml.com>
Date: Thu, 19 Feb 2026 17:09:11 +0000
Subject: [PATCH 07/10] Update Pipewire to 1.6.7

---
 elements/components/pipewire-base.bst | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/elements/components/pipewire-base.bst b/elements/components/pipewire-base.bst
index a09759fdf..9b4ee6c76 100644
--- a/elements/components/pipewire-base.bst
+++ b/elements/components/pipewire-base.bst
@@ -56,6 +56,10 @@ variables:
     -Dlibffado=disabled
     -Dsdl2=disabled
     -Dsnap=disabled
+    -Dfftw=disabled
+    -Dbluez5-codec-ldac-dec=disabled
+    -Dbluez5-plc-spandsp=disabled
+    -Donnxruntime=disabled
 
 config:
   install-commands:
@@ -142,4 +146,5 @@ public:
 sources:
 - kind: git_repo
   url: freedesktop:PipeWire/pipewire.git
-  ref: 1.4.9-0-gfd60e04525f3a04d90bf50085222e0cc9139b4a4
+  track: "*.*.*"
+  ref: 1.6.8-0-gb741e0c74f5436f0c925f7741140db0efd32cf4e
-- 
2.54.0

